### PR TITLE
Split terraform state into mutiple layers

### PR DESCRIPTION
**Pull request by sanjeevkopsai**

### CODE REVIEW

File: level1/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    value = aws_subnet.private[*].id
+}
+
+output "vpc_id" {
+    value = aws_vpc.main.id
+}
+
+output "vpc_cidr" {
+    value = var.vpc_cidr
+}
```
Comment (geoffrey1330) [inline level1/outputs.tf:15]: could you please, add newline at the end of file.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: level0/provider.tf [hunk 2]
```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+  region = "us-west-1"
+}
```
Comment (geoffrey1330) [inline level0/provider.tf]: could you please, add newline at the end of file.

File: level1/outputs.tf (hunk 1 above)
Comment (geoffrey1330) [inline level1/outputs.tf]: could you please, add newline at the end of file.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: level0/provider.tf (hunk 2 above)
Comment (sanjeevkopsai) [reply level0/provider.tf]: done - please review

File: level1/outputs.tf (hunk 1 above)
Comment (sanjeevkopsai) [reply level1/outputs.tf]: done

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**